monitoring/monitors/kube/podMonitor-eventrouter.yaml

```diff
@@ -1,15 +1,15 @@
 apiVersion: monitoring.coreos.com/v1
 kind: PodMonitor
 metadata:
-  name: eventrouter
+  name: v4m-eventrouter
   labels:
```

Comment:
In an UIP scenrario, do we need to worry about removing the existing podMonitors tied to the previous instance of Event Router (i,.e. the one in kube-system)?

Reply:
After our conversations, I don't think it's necessary to make this change.
I'm reverting it for now and will test it.